Header reputation and network info styling

dit7ya opened the pull request:
## Description


**New stuff** ✨

Added hover state and tooltip to the current user reputation in the nav bar.
![Colony](https://user-images.githubusercontent.com/14034137/151590265-e73b4b9b-3f46-49cf-b9db-cfb0486840c4.png)

![Colony (2)](https://user-images.githubusercontent.com/14034137/151592844-705131ca-aa65-4f06-806f-0e0f6eb12dfb.png)


Also added hover state to the network info in the nav bar since that was missing too.

![Colony (1)](https://user-images.githubusercontent.com/14034137/151590817-d57fc72d-7c92-443a-a963-618236e5f5d7.png)


**Changes** 🏗

Switched the star icon to the right in MemberReputation component. This affects all places where the MemberReputation component is.

**Notes**

Used a slightly different copy than the spec to avoid introducing a new variable in the MemberReputation component. (Displaying the reputation percentage in the tooltip is complicated since the value of that is computed in the MemberReputation component, which is a child of the Tooltip component.

Resolves  #3129.

ArmandoGraterol reviewed (commented):
Nice! Looks good right now but I think it would look better if the tooltip was lower, so that it doesn't overlaps the reputation.

The `Tooltip` component can receive a prop to move the tooltip around like this:
```
popperProps={{
  modifiers: [
    {
      name: 'offset',
      options: {
        offset: [-8, 8],
      },
    },
  ],
}}
```

dit7ya commented:
@ArmandoGraterol Thank you for pointing it out. I have added `popperProps` (but slightly changed it since I also changed the placement of the tooltip to be `bottom-start`.

ArmandoGraterol reviewed (approved):
Nice work! It looks good to me

chinins reviewed (approved):
Looks good!

arrenv reviewed (changes requested):
@dit7ya Are you able to take a look at the alignment of the text in the box, to vertically align it better. I am creating another issue regarding the alignment and font sizes difference in the other elments.

![header-text-alignment](https://user-images.githubusercontent.com/33682027/151947690-bda7d6ea-7873-4fb4-8be9-06d211228005.png)


Also, the hover seems a little temperamental.

![tempermental-on-hover](https://user-images.githubusercontent.com/33682027/151947591-e471d34c-d030-4c4e-8ba4-79f40e8f8858.gif)

dit7ya commented:
Good point @arrenv. After some tinkering, I think the visible rerender of the tooltip is due to the existing title property of the `Icon` component inside it. Can you please confirm if you observe the same behavior in the Lock icon in the Events page? I guess the question mark tooltip does not have this problem because it is circular. I don't really know though. Removing the Icon from the component makes the rendering much more predictable.

If that is the case, the only solution I can think of is re-writing the Icon component to make the title optional. What do you think?

dit7ya commented:
I can't seem to get around the shaky tooltip behavior even after removing the icon or the text. Any suggestions @arrenv ?

arrenv reviewed (changes requested):
@dit7ya I think that is not really a big issue.

However, I did notice the title tag overlapping the popover. I think that it is worth adding another component prop to disable the title tag to account for this.

![title-tag-overlay](https://user-images.githubusercontent.com/33682027/152964783-832eb639-9f47-45a6-8ea5-3817fb558d8c.jpg)
.

dit7ya commented:
@arrenv Made the title property optional in the Icon component and removed it from the tooltiped component. Also, I don't think this bug is related to this PR but please take a look at the console when you hover over the reputation component. It is creating  failing network requests which might be a separate issue somewhere else.
